pkg/modules/resiliency_test.go

```diff
@@ -16,14 +15,62 @@ package modules
 import (
 	"context"
 	"testing"
-
 	csmv1 "github.com/dell/csm-operator/api/v1"
 	drivers "github.com/dell/csm-operator/pkg/drivers"
 	utils "github.com/dell/csm-operator/pkg/utils"
 	"github.com/stretchr/testify/assert"
 	rbacv1 "k8s.io/api/rbac/v1"
+	applyv1 "k8s.io/client-go/applyconfigurations/apps/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+	ctrlClient "sigs.k8s.io/controller-runtime/pkg/client"
+	ctrlClientFake "sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
+func TestResiliencyInjectDeployment(t *testing.T) {
```

Comment:
We have greenfield injection test, can we add brownfiled injection test case too ?